### DISCUSSION TITLE
Adding PowerAutomate flows

### DISCUSSION THREAD

**Discussion by MACAD2023** [community/community#87712] [New to GitHub] [answered]
Hi, I'm completely new one here. I'd like to share with community my flows, but in fact I don't know how ? Can someone explain me (and other) how to manage this ?
Thank you.

**Comment by queenofcorgis** [comment]
Hi @MACAD2023  , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by MACAD2023** [reply]
Hi @queenofcorgis !! I would like to share my flows, but I don't found how to do that. Maybe you could explain me ?

**Reply by ghost** [reply]
Flows in sense do you mean work flows in github or powerautomate  flows of Microsoft...

**Reply by ghost** [reply]
Thats what @queenofcorgis is asking, what is the thing you need to share...

**Reply by MACAD2023** [reply]
@Yuheshpandian PowerAutomate flows :-)

**Reply by ghost** [reply]
You could export the flows as .zip file then add it to a repository then share the link with your community

**Reply by ghost** [reply, marked as answer]
Add the exported .zip file to your GitHub repository.
You can do this by using the GitHub web interface or by using Git commands locally.

**Reply by ghost** [reply]
Refer this video [link](https://youtu.be/eCDL1Ix3aGY?si=HERHI73PciC_Oyyv) Top export it as .zip file...

**Reply by MACAD2023** [reply]
Thank you !! Soon I'll share my first simple flows.

**Reply by ghost** [reply]
All the best
